app/helpers/remote_execution_helper.rb

```diff
@@ -30,15 +30,17 @@ def job_invocation_chart(invocation)
   end
 
   def job_invocation_status(invocation)
-    if invocation.task.blank?
-      _('Job not started yet 0%')
-    elsif invocation.task.state == 'scheduled'
-      _('Job set to execute at %s') % invocation.task.start_at
-    elsif invocation.task.state == 'stopped' && invocation.task.result == 'error'
-      invocation.task.execution_plan.errors.map(&:message).join("\n")
-    else
-      label = invocation.task.pending ? _('Running') : _('Finished')
-      label + ' ' + (invocation.task.progress * 100).to_i.to_s + '%'
+    case invocation.status
+      when HostStatus::ExecutionStatus::QUEUED
+        _('queued')
+      when HostStatus::ExecutionStatus::RUNNING
+        _('running %s%') % invocation.progress
+      when HostStatus::ExecutionStatus::OK
+        _('succeeded')
+      when HostStatus::ExecutionStatus::ERROR
+        _('failed')
+      else
+        _('unknown status')
```

Comment:
hm, I wanted to be consistent with time info we display in start column but if you want I can change it back to capital letters, not a big deal :-) also the percentage will hopefully be later replaced by some small chart (as it was in wireframes)